src/components/Dropdown/Dropdown.tsx

```diff
@@ -58,32 +60,14 @@ export class Dropdown extends BaseComponent<IDropdownProps, any> {
     });
   }
 
-  public componentWillUpdate(nextProps: IDropdownProps, nextState: IDropdownState) {
-    if (this.state.isOpen !== nextState.isOpen) {
-      if (nextState.isOpen) {
-        this._events.on(window, 'focus', this._onFocusChange, true);
-      } else {
-        this._events.off();
-      }
-    }
-  }
-
-  public componentDidUpdate(prevProps: IDropdownProps, prevState: IDropdownState) {
-    if (prevState.isOpen === false && this.state.isOpen === true) {
-      this._scrollOnOpen();
-    } else if (prevState.selectedIndex !== this.state.selectedIndex) {
-      this._scrollSelectedItemIntoView();
-    }
-  }
-
   public render() {
     let { label, options, onRenderItem = this._onRenderItem } = this.props;
     let { id, isOpen, selectedIndex, isDisabled } = this.state;
     let selectedOption = options[selectedIndex];
 
     // Need to assign role application on containing div because JAWS doesnt call OnKeyDown without this role
     return (
-      <div ref='root'>
+      <div role='application' ref='root'>
```

Comment:
Is role="application" really needed? Based on role=combobox, the screen reader will automatically switch based on user settings. We should avoid hijacking control when we can.